lessons/operator/02/README.md

```diff
@@ -1,257 +1,267 @@
+- [EXERCISE 1: Install the Workshop](#exercise-1-install-the-workshop)
+- [Discussion](#discussion)
+- [Learn More](#learn-more)
+- [Next Steps](#next-steps)
+
+## Goals
+
+In this lesson we will setup a local multi-node Sensu environment for use during the workshop.
+This lesson is intended for operators of Sensu or anyone who would like to explore Sensu from a technical perspective.
+
+## Prerequisites
+
+This workshop was designed with common IT/DevOps/SRE tooling in mind, so we expect most users will already have everything they need installed. 
+All you need to get started is a laptop or workstation with a `git` client, [Docker](https://www.docker.com/), text editor, and a modern web browser.
+
+Please review this list of prerequisites to make sure your system is ready to go:
+
+1. **Workstation**
+
+   Workshop users will need a 64-bit workstation running a recent version of **macOS, Windows, or Linux**.
+ 
+1. **Docker**
+   - Mac users should install [Docker CE Desktop for Mac](https://hub.docker.com/editions/community/docker-ce-desktop-mac).
+   - Windows users should install [Docker CE Desktop for Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows).
+   - Linux users should follow the [Docker Engine installation guide](https://docs.docker.com/engine/install/).
+
+1. **Docker Compose (`docker-compose`)**
+
+   To install `docker-compose`, please read the ["Install Docker Compose"](https://docs.docker.com/compose/install/) documentation.
+
+1. **Git Client (`git`)**
+
+   To install `git`, please read the [`git` downloads](https://git-scm.com/downloads) page.
+   
+1. **Supported Web Browser**
+
+   The workshop expects that you have a modern web browser installed. 
+   Please verify that you have a recent version of one of these common browsers:
+   - [Chrome](https://www.google.com/chrome/fast-and-secure/)
+   - [Safari](https://support.apple.com/en-us/HT204416)
+   - [Firefox](https://www.mozilla.org/en-US/firefox/new/)
+   - [Microsoft Edge](https://www.microsoft.com/en-us/edge)
+   
+1. **Compatible Text Editor**
+
+   This workshop involves creating and editing plain-text configuration files and monitoring code templates.
+   We suggest using a text editor such as:
+   - [Microsoft Visual Studio Code](https://visualstudio.microsoft.com)
+   - [Atom](https://atom.io)
+   - [Sublime Text](https://www.sublimetext.com)
+   - [`vim`](https://www.vim.org/download.php)
+   - [`emacs`](https://www.gnu.org/software/emacs/download.html)
+   
+   _NOTE: Windows users should avoid using `Notepad.exe` which can save edited files in incompatible encodings._
+   
+1. **Optional CLI Tools**
+
+   The workshop includes examples using some optional CLI utilties:
+   - `jq` ([website](https://stedolan.github.io/jq/), [downloads](https://stedolan.github.io/jq/download/))
+
+
+## The Workshop Environment
+
+To fully demonstrate Sensu's capabilities, it is necessary to have a multi-node environment.
```

Comment:
s/it is necessary to have a multi-node environment/it is helpful to emulate one of many common reference architectures in which Sensu is commonly deployed/ ??